website/content/docs/agent/template.mdx

```diff
@@ -187,8 +187,11 @@ can be used here:
   to "true". Also see [`exit_on_retry_failure` in global Vault Agent Template Config](/docs/agent/template#interaction-between-exit_on_retry_failure-and-error_on_missing_key).
 - `exec` `(object: optional)` - The exec block executes a command when the
   template is rendered and the output has changed. The block parameters are
-  `command` `(string slice: required)` and `timeout` `(string: optional, defaults
-  to 30s)`.
+  `command` `(string or array: required)` and `timeout` `(string: optional, defaults
+  to 30s)`. `command` can be given as a string or array of strings to execute, such as
+  `"touch myfile"` or `["touch", "myfile"]`. We strongly recommend using an array of strings,
```

Comment:
It's always nice when giving recommendations to provide a justification.  I think the danger we're protecting against is called "command injection", so you could say "To protect against command injection, we strongly recommend..."

Reply:
Yeah, that makes sense. This way we also avoid splitting up strings if they contain commas, too, which felt a little weirder to point out. Thanks for the suggestion, will update accordingly